AppAuth-JS/built/types.d.ts

```diff
@@ -0,0 +1,27 @@
+export interface StringMap {
```

Comment:
This is generated source file. All the files in built/ folder are generated JS file for the Google's lib fork